resources/glossary/links.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 objectId: 2f3e8d88-0205-4043-f46f-0ba3191t4626
 title: Links
-description: Links between different entities represent the relationships and connections between them.
+description: "Links between different entities represent the relationships and connections between them."
```

Comment:
We shouldn't need these – is there somewhere that missing them is causing an issue? Some of the other existing pages do not have them. FWIW I slightly prefer not having them as it's cleaner.

Reply:
Addressed in https://github.com/hashintel/hash/pull/669/commits/2e9a9e6ecf0ba413bc85f78ad80eed5db76b1b60